lms/djangoapps/courseware/views/views.py

```diff
@@ -1911,7 +1911,7 @@ def financial_assistance_request(request):
         goals = data['goals']
         effort = data['effort']
         marketing_permission = data['mktg-permission']
-        ip_address = get_ip(request)
+        ip_address, _ = get_client_ip(request)
```

Comment:
```suggestion
        ip_address = get_client_ip(request)[0]
```
Keep this consistent unless there is a reason not to.